addons/website/static/src/snippets/s_website_form/options.js

```diff
@@ -1704,6 +1778,55 @@ options.registry.WebsiteFieldEditor = FieldEditor.extend({
     _getSelect: function () {
         return this.$target[0].querySelector('select');
     },
+    /**
+     * Generates error message for requirement set on field if validation fails.
+     *
+     * @private
+     * @param {string} comparator The way that message needs to be formed.
+     * @param {string} [condition] Expected value of the field
+     * @param {string} [between] The maximum date value in case comparator
+     *      is between or !between
+     * @returns {string} Default error message
+     */
+    _defaultMessage(comparator, condition, between, type) {
+        const textMessages = {
+            "contains": "This field must include keyword %s.",
+            "!contains": "This field must not include keyword %s.",
+            "substring": "This field must include keyword %s.",
+            "!substring": "This field must not include keyword %s.",
+            "greater": "Invalid: field is not greater than %s.",
+            "less": "Invalid: field is not less than %s.",
+            "greater or equal": "Invalid: field is not greater than or equal to %s.",
+            "less or equal": "Invalid: field is not less than or equal to %s."
+        };
+        
+        if (textMessages[comparator]) {
+            return _t(textMessages[comparator], condition);
```

Comment:
Translation will not work, you need to translate terms while defining the message above.